### DISCUSSION TITLE
Could I continue to train when adding new data into trian dataset

### DISCUSSION THREAD

**Discussion by fgbxx** [MIC-DKFZ/nnUNet#778] [Q&A] [answered]
hi, Fabian
   I used nnunet to train model, but my label data may increase iterately.
   My question is  , should I must re-train model or can continue to train based on pre-trained model?
   I just test, learning rate reach to 0 when I change the max epoches to 2000(the last model had been trained 1000 epoches)

Best wishes, 
Gang

**Comment by FabianIsensee** [comment, marked as answer]
Hi, you can load a pretrained model at the start of the training, have a look at `nnUNet_train -h`. This pretrained model has to have been trained with the same plans. Also beware of the data splits. If you add new training cases you need to remove the old splits_final.pkl file, otherwise they will not be used! You also need to ensure not to break your previous cross-validation splits
